task/dev3/Operations/MostPopularName.java

```diff
@@ -0,0 +1,41 @@
+package task.dev3.Operations;
+
+import task.dev3.User.*;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Do operations for search name.
+ */
+public class MostPopularName {
+  public final String WOMAN = "woman";
+  
+  /**
+   * Search the most popular womens name.
+   * @param userList - list contains all added users.
+   * @return String - the most popular female name.
+   */  
+  public String mostPopularFemaleName(List<User> userList) {
```

Comment:
it's not flexible. if you want to have the same but for men you have to make codechanges here or have a big copy-pasted snippet